pkgs/development/python-modules/diffimg/default.nix

```diff
@@ -21,7 +22,10 @@ buildPythonPackage rec {
   # fix offered to upstream https://github.com/nicolashahn/diffimg/pull/6
   postPatch = ''
     substituteInPlace diffimg/test.py \
-      --replace "from diff import diff" "from diffimg.diff import diff"
+      --replace-warn "from diff import diff" "from diffimg.diff import diff"
+  '' + lib.optionalString (pythonAtLeast "3.12") ''
+    substituteInPlace diffimg/test.py \
+      --replace-warn "3503192421617232" "3503192421617233"
```

Comment:
I'd like to see upstream at least being notified of this problem (even though upstream don't seem particularly responsive) to check others are seeing the same problem.

Reply:
good point, issue opened https://github.com/nicolashahn/diffimg/issues/7